modules/database/variables.tf

```diff
@@ -65,3 +65,10 @@ variable "sort_buffer_size" {
   type        = number
   default     = 262144
 }
+
+
+variable "ssl_connection_string" {
```

Comment:
Let's change this to `"force_ssl"`.  Description should be "Enforce SSL connections via the Cloud SQL Proxy".  Eventually we'll probably want to wire this through the DB instance itself so that flag is set.